pkg/printers/internalversion/describe.go

```diff
@@ -1076,6 +1076,16 @@ func printAzureFilePersistentVolumeSource(azureFile *api.AzureFilePersistentVolu
 		azureFile.SecretName, ns, azureFile.ShareName, azureFile.ReadOnly)
 }
 
+func printFlexPersistentVolumeSource(flex *api.FlexPersistentVolumeSource, w PrefixWriter) {
```

Comment:
interesting, there is never a unit test for flex

Reply:
and secrets injection is not tested anywhere